README.md

```diff
@@ -7,20 +8,43 @@ This GitHub Action (created using typescript) uses [CLI for Microsoft 365](https
 
 ## Usage
 ### Pre-requisites
+
 Create a workflow `.yml` file in your `.github/workflows` directory. An [example workflow](#example-workflow---cli-for-microsoft-365-login) is available below. For more information, reference the GitHub Help Documentation for [Creating a workflow file](https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file).
 
 ### Inputs
-- `ADMIN_USERNAME` : **Required** Username (email address of the admin)
-- `ADMIN_PASSWORD` : **Required** Password of the admin
+
+- `ADMIN_USERNAME` : Username (email address of the admin)
+- `ADMIN_PASSWORD` : Password of the admin
+- `CERTIFICATE_ENCODED` : Base64-encoded string with certificate private key
+- `CERTIFICATE_PASSWORD` : Password for the certificate
+- `AAD_APP_ID` : App ID of the Azure AD application to use for certificate authentication
```

Comment:
Let's align this with the option name and description from `login` command for consistency.

```suggestion
- `APP_ID`: App ID of the Azure AD application to use for authentication. If not specified, use the app specified in the `CLIMICROSOFT365_AADAPPID` environment variable. If the environment variable is not defined, use the multitenant PnP Management Shell app
```